Makefile

```diff
@@ -147,6 +147,7 @@ clean:
 cleanall: clean 
 	@echo -e ${GREEN}Performing Clean on frameworks [$(UT_CORE_DIR)/framework]${NC}
 	@$(RM) -rf $(UT_CORE_DIR)/framework
+	@$(RM) -rf $(BIN_DIR)
```

Comment:
U won’t be able todo that specially you’ll have to remove .so &.a I think some of the tests place files required in the bin directory therefore your going to kill the scripts

Reply:
got it. Working on it.